src/components/Header.js

```diff
@@ -47,6 +47,10 @@ class Header extends Component {
               <div className="title-container">
                 <HeaderTitleTypeAnimation />
               </div>
+              <div id="slogan">
+                <p>Creating Apps for caring life changers.</p>
```

Comment:
yeah, I also feel that it looks packed! Also, we shouldn't mix the flat design that's already on the page with some blurry and shadowy stuffz (but good work to play around with it ;)

We can leave it for now and think of a more spectacular place to build it in? Like it's own white-spaced container, maybe with a different color?

Reply:
![Peek 2021-05-26 14-07](https://user-images.githubusercontent.com/11299099/119657127-cdc45780-be2b-11eb-8d4f-34548a1d4bc4.gif)

I find it here nicer then on the first page. Also benefit: no need for adjusting the css for mobile what slowly but steadily develops into time consuming. I would fiddle around with it, but I really like it more this way.

Reply:
Really love that solution!